charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -22,6 +22,17 @@ alfresco-process-services:
       memory: "2000Mi"
     limits:
       memory: "3000Mi"
+  processWorkspace:
+    ingress:
+      path: "/apw"
+    environment:
+      APP_CONFIG_BPM_HOST: "http://alfresco-process.REPLACEME"
+      APP_CONFIG_ECM_HOST: "http://alfresco-process.REPLACEME"
+      APP_CONFIG_AUTH_TYPE: "OAUTH"
+      APP_CONFIG_OAUTH2_HOST: "http://alfresco-identity-service.REPLACEME/auth/realms/alfresco"
+      APP_CONFIG_OAUTH2_CLIENTID: "alfresco"
+      APP_CONFIG_OAUTH2_REDIRECT_LOGIN: "/apw/"
```

Comment:
should we also consider configuring APP_CONFIG_OAUTH2_REDIRECT_LOGOUT?

Reply:
Indeed, good catch.  I had that locally (and the team had it on the previous branch) but looks like it didn't make it into the commit.

Added.